.env.example

```diff
@@ -1,7 +1,15 @@
-# MATH Package
-DEV_MONGO_URI=mongodb://127.0.0.1:27017/booking
-DEV_MONGO_USER=booking-user
-DEV_MONGO_PASS=booking-pass
+PREFFIX_MONGO_AUTH_SOURCE=db where you can authenticate when you don't have any information in your db
+PREFFIX_MONGO_URI=mongodb://127.0.0.1:27017/dbName
+PREFFIX_MONGO_USER=
+PREFFIX_MONGO_PASSWORD=
+// Using mongo shared connection is going to ask for a prefix.
+// this in uppercasewill be your preffix in you env_vars
```

Comment:
Can you replace `preffix` with `prefix` (with just an `f`) everywhere you wrote it?